sqrl-cli/src/test/java/com/datasqrl/packager/config/ServerConfigTemplateTest.java

```diff
@@ -59,7 +61,10 @@ public static void main(String[] args) {
     var afterParsing = new ServerConfig(new JsonObject(original));
 
     if (!Objects.equals(original, mapper.convertValue(afterParsing, Map.class))) {
-      mapper.copy().writerWithDefaultPrettyPrinter().writeValue(TEMPLATE, afterParsing);
+      mapper
+          .configure(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY, true)
+          .writerWithDefaultPrettyPrinter()
+          .writeValue(TEMPLATE, afterParsing);
```

Comment:
These calls will modify the original object, I'd do a `.copy()` before, cause `SqrlObjectMapper` is used elsewhere too, and calling this will change its behavior afterwards.

Reply:
That never going to cause a problem, as it a separated main()